fakeredis/_server.py

```diff
@@ -2324,13 +2332,18 @@ def _lua_redis_pcall(self, lua_runtime, expected_globals, op, *args):
 
     def _lua_redis_log(self, lua_runtime, expected_globals, lvl, *args):
         self._check_for_lua_globals(lua_runtime, expected_globals)
-        if len(args) == 1:
-            msg = args[0]
-            LOGGER.log(lvl or logging.INFO, msg.decode('utf-8') if isinstance(msg, bytes) else msg)
+        if len(args) < 1:
+            raise redis.ResponseError(REQUIRES_MORE_ARGS.format("redis.log()", "two"))
+        if lvl not in REDIS_LOG_LEVELS.values():
+            raise redis.ResponseError(LOG_INVALID_DEBUG_LEVEL)
+        msg = ' '.join([x.decode('utf-8')
+                        if isinstance(x, bytes) else str(x)
+                        for x in args if not isinstance(x, bool)])
+        LOGGER.log(REDIS_LOG_LEVELS_TO_LOGGING.get(lvl, logging.WARNING), msg)
```

Comment:
Is it necessary to use `get` rather than just `[]`, given that you've already checked that `lvl` is in `REDIS_LOG_LEVELS.values()`?